src/schedlib/policies/lat.py

```diff
@@ -570,32 +627,32 @@ def init_seqs(self, t0: dt.datetime, t1: dt.datetime) -> core.BlocksTree:
             is_leaf=lambda x: isinstance(x, dict) and 'type' in x
         )
 
+        # get cal targets
+        if cfile is not None:
+            cal_targets = inst.parse_cal_targets_from_toast_lat(cfile)
+            cal_targets[:] = [cal_target for cal_target in cal_targets if cal_target.t0 >= t0 and cal_target.t0 < t1]
+
+            for i, cal_target in enumerate(cal_targets):
+                if self.elevations_under_90 and cal_target.el_bore > 90:
+                    cal_targets[i] = replace(cal_targets[i], el_bore=180-cal_targets[i].el_bore)
```

Comment:
I think this needs to have elevations over 90 not implemented. (And/or we need the same flipping I mentioned here: https://github.com/simonsobs/scheduler/pull/239#issuecomment-2877187231)